shared/params.go

```diff
@@ -141,6 +141,22 @@ func (p ProductSpec) Matches(run TestRun) bool {
 	return true
 }
 
+// DisplayName returns a capitalized version of the product's name.
```

Comment:
Do we have enough info to include the channel name here, or at the call site?

Reply:
Yes, the channel is available in the label set. I'd do that in a separate PR, though.

Reply:
SGTM.